kolibri/plugins/coach/assets/src/views/home/HomePage/LessonsBlock.vue

```diff
@@ -15,6 +15,7 @@
         :name="tableRow.name"
         :tally="tableRow.tally"
         :groupNames="tableRow.groups"
+        :to="classRoute('ReportsLessonReportPage', { lessonId: tableRow.key })"
```

Comment:
for consistency with quizzes, maybe this should link to `ReportsLessonLearnerListPage`